src/components/atoms/button/button.scss

```
@@ -0,0 +1,27 @@
+@use "../../variables" as *;
+
+.button {
+  color: $white;
+  background-color: $black;
+  transition: 0.2s ease;
+  width: 200px;
+  height: 55px;
+  text-align: center;
+  vertical-align: middle;
+  display: table-cell;
```

```suggestion
  justify-content: center;
  align-items: center;
  display: flex;
```
table-cellは[table](https://saruwakakun.com/html-css/basic/table)に使うものなので、基本的にそれ以外では使わないようにしましょう。

今回のように要素を中央に揃える場合など、子要素の配置を制御したい場合はdisplay: flex;を使うことが多いです。
https://webdesign-trends.net/entry/8148